### PR TITLE
Feature/custom root

### PR DESCRIPTION
**Pull request by afaikafk**
Add additional `fsal.chroot` config param to specify an optional, common root folder under all base paths.

### CODE REVIEW

File: fsal/fsdbmanager.py [hunk 1]
```diff
@@ -74,7 +74,13 @@ class FSDBManager(object):
     SLEEP_INTERVAL = 0.500
 
     def __init__(self, config, context):
-        paths = config['fsal.basepaths']
+        chroot = config.get('fsal.chroot') or ''
```
Comment (foxbunny) [inline fsal/fsdbmanager.py:77]: `chroot = config.get('fsal.chroot', '')` should be sufficient here, I think.

File: fsal/fsdbmanager.py [hunk 2]
```diff
@@ -74,7 +74,13 @@ class FSDBManager(object):
...
+        if chroot:
+            logging.debug(u'chroot to: %s', chroot)
```
Comment (foxbunny) [inline fsal/fsdbmanager.py]: Log messages are capitalized in the other parts of the code base.

File: fsal/fsdbmanager.py (hunk 1 above)
Comment (afaikafk) [inline fsal/fsdbmanager.py:77]: This way it covers the case of `None`

**Comment by foxbunny** [comment]
Looks good to me save for minor nitpicks.

**Pull request merged**